tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1245,6 +1245,8 @@ Status StorageManager::get_fragment_info(
     if (!sparse)
       array_schema->domain()->expand_to_tiles(&expanded_non_empty_domain);
 
+    meta->load_rtree(encryption_key);
```

Comment:
This will be super slow for all scenarios where the user wants to see the fragment information, without the R-trees. Loading the R-tree of a fragment needs to happen conditionally when the new API is called and only for the specific fragment that it is requested.